codemon/CodemonHelp.py

```diff
@@ -1,11 +1,20 @@
 from clint.textui import colored
 
 def showHelp():
+	print()
 	print("           ---CODEMON---          ")
+

```

Comment:
We don't need spaces on every line here. The previous version was okay, so please revert back.